src/bin/bench-tps.rs

```diff
@@ -401,31 +402,24 @@ fn should_switch_directions(num_tokens_per_account: i64, i: i64) -> bool {
 fn main() {
     logger::setup();
     metrics::set_panic_hook("bench-tps");
-    let mut threads = 4usize;
-    let mut num_nodes = 1usize;
-    let mut time_sec = 90;
-    let mut sustained = false;
-    let mut tx_count = 500_000;
 
     let matches = App::new("solana-bench-tps")
         .version(crate_version!())
         .arg(
             Arg::with_name("network")
                 .short("n")
                 .long("network")
-                .value_name("HOST:PORT")
+                .value_name("IP:PORT")
```

Comment:
I noticed this api which is built-in which could resolve hostnames: https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html maybe a change for a different PR though. Have you seen it?

Reply:
Yeah, I've seen ToSocketAddrs (last week spent a lot of time with it ;) ) .  Accepting a string here pushes failure to find_leader(), which would have to offer better failure modees, or I can do DNS here, which may take time.  I'll change it to HOST:PORT and see how we do.